clang/lib/CConv/RewriteUtils.cpp

```diff
@@ -149,375 +148,37 @@ void GlobalVariableGroups::addGlobalDecl(VarDecl *VD,
 }
 
 std::set<VarDecl *> &GlobalVariableGroups::getVarsOnSameLine(VarDecl *VD) {
-  assert (globVarGroups.find(VD) != globVarGroups.end() &&
-         "Expected to find the group."); return *(globVarGroups[VD]);
+  assert (GlobVarGroups.find(VD) != GlobVarGroups.end() &&
+         "Expected to find the group.");
+  return *(GlobVarGroups[VD]);
```

Comment:
Was there no `return` before? Perhaps this was a latent source of bugs, if so.

Reply:
There was just a line break missing between the assert and the return